### PR TITLE
Added the purchase option

### PR DESCRIPTION
**Pull request by athiruma**
1. Added the purchase type of cost-explorer.
2. Purchase_type filter gives the cost details of AWS ec2  plans. i.e Savings Plan, Spots, On Demand, Reservations.

### CODE REVIEW

**Comment by ebattat** [comment]
@athiruma, pls details the types in the description

**Comment by athiruma** [comment]
> @athiruma, pls details the types in the description

added

File: jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py [hunk 1]
```diff
@@ -29,8 +29,7 @@
 es_index_psap = 'cloud-governance-cost-explorer-psap'
 es_index_perf_scale = 'cloud-governance-cost-explorer-perf-scale'
 es_index_global = 'cloud-governance-cost-explorer-global'
-
-cost_tags = ['ChargeType', 'User', 'Budget', 'Project', 'Manager', 'Owner', 'LaunchTime', 'Name', 'Email', 'Environment']
+cost_tags = ['PurchaseType', 'ChargeType', 'User', 'Budget', 'Project', 'Manager', 'Owner', 'LaunchTime', 'Name', 'Email', 'Environment']
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:32]: Why we need a tag for PurchaseType, is it not like ec2 type data ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:32]: No, it represents the EC2 cost plans

Comment (ebattat) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:32]: Why we need to tag it on every ec2 resource ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:32]: No, it's not the tag. It is the cost explorer filter dimension.

Comment (ebattat) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:32]: Ok

**Pull request merged**